packages/connect/CHANGELOG.md

```diff
@@ -10,6 +10,23 @@
 
 Use the persistent link [connect.trezor.io/9](https://connect.trezor.io/9/) to access the latest stable version of Connect Explorer.
 
+# 9.4.6-beta.1
+
+This release adds apiType to transports and blocks to blockchainUnsubscribe.
```

Comment:
this is uninteresting for the 3rd parties. maybe the blocks unsubscribe is important but the sentence `and blocks to blockchainUnsubscribe.` doesn't explain anything